src/index.js

```diff
@@ -1 +1,29 @@
-import './style.scss';
\ No newline at end of file
+import './style.scss';
+import newGameID from './modules/newGameID';
+import saveScore from './modules/saveScore';
+import successOrErrorMessage from './modules/succesOrErrorMessage';
+import renderScores from './modules/renderScores';
+
+const checkGameID = () => {
+  if (window.localStorage.getItem('Game ID')) {
+    return JSON.parse(window.localStorage.getItem('Game ID'));
+  }
+  return newGameID('TiagosGame');
+};
+const gameID = checkGameID();
+const refreshBtn = document.getElementById('refresh');
+const userName = document.getElementById('name');
+const userScore = document.getElementById('score');
+const submitBtn = document.getElementById('submit-btn');
+
+submitBtn.addEventListener('click', () => {
+  const apiReturn = saveScore(userName.value, userScore.value, gameID);
+  successOrErrorMessage(apiReturn);
+});
+
+refreshBtn.addEventListener('click', () => {
+  renderScores();
+});
+
+// JSON.parse()
+// JSON.stringify()
```

Comment:
- [ ] OPTIONAL: Your data is only loading on clicking to the refresh button. It would be nice if the user can see all the data on the page load for a better user experience. You can add to also improve your code quality with a very UX.

Hint: You can use a window listener method to fetch data on page load or refresh.
```
  // on windows load
    window.addEventListener('load', () => {
        renderScores();
    }, false);
```

Reply:
Hi @Kandy-Peter , thank's a lot for your review. I will do the suggested improvement (window load) and also I'm sorry about not pointing it on the readme file, I was going to put it there and also on the submission form, but I forgot,